main.lua

```diff
@@ -159,6 +158,9 @@ function _load(filename)
 	else
 		setfps(30)
 	end
+
+	-- We don't want the first frame's dt to include time taken by love.load.
+	if love.timer then log(love.timer.step()) end
```

Comment:
This line is actually from LOVE and it should probably stay in the love.run function (unless we use love.timer's delta time somewhere else, I don't see what value stepping the timer here does).

Reply:
This here because I think we need to do it after every reload for the same reason it was done once

Reply:
A better way would be to have an event or some way to communicate it to love but for this I need some help

Reply:
Actually, since this gets called at the end of loading, I guess it can stay here if it needs to be here.